api/tests/Feature/PoolCandidateTest.php

```diff
@@ -411,7 +411,12 @@ public function testFilterByEmploymentEquity(): void
       }
     ', [
       'where' => [
-        'isIndigenous' => true,
+        'equity' => [
```

Comment:
Try adding a test or two where the fields are missing instead of just false.  The schema has them as nullable, after all.  You'll find that you need to add some null handling. :grin:
![image](https://user-images.githubusercontent.com/8978655/172181855-7195e386-191d-4a6d-8465-e0e1031e1889.png)